switching/input/messages/C1.py

```diff
@@ -213,6 +213,8 @@ def data_vigor(self):
             val = self.pm.FechaVigor.text
         except AttributeError:
             pass
+        if val in INVALID_DATES:
```

Comment:
To validate the date, is better a to date string conversion in a try/catch block. If it fails, return None. Better in function to encapsulate the logic